src/composables/useSizes/index.ts

```diff
@@ -0,0 +1,30 @@
+import { computed, readonly } from 'vue'
+import type { MaybeRefOrGetter, MaybeRef, ComputedRef, Ref } from 'vue'
+import { refDebounced, toValue, useElementSize, useWindowSize } from '@vueuse/core'
+
+export interface SizesType {
+  height: Readonly<Ref<number>>
+  width: Readonly<Ref<number>>
+  aspectRatio: ComputedRef<number>
+}
+
+export default function useSizes(
+  windowSize: MaybeRefOrGetter<boolean>,
+  canvas: MaybeRef<HTMLCanvasElement>,
+  debounce: number = 10,

```

Comment:
A naming suggestion. 

Background: Vueuse calls this argument `ms`. 

Maybe we call it `debounceMs`? That makes it clear that it's expecting millisecond units, and not e.g., fractions of a second or a boolean.

Reply:
Oh, I see, that makes sense. I should indeed give them more accurate names. I'll go make the changes.